test_component.py

```diff
@@ -63,9 +63,10 @@ def oncounter(counter, id, type):
 
             # CALL
             try:
-                res = yield self.call(u'com.example.add', x, 3)
+                res = yield self.call(u'com.example.add2', x, 3)
```

Comment:
This conflicts because if the python component registers com.example.add2, then the Java component cannot register that name. So we need that to be distinct.